core/src/main/java/org/apache/carbondata/core/datastore/filesystem/AbstractDFSCarbonFile.java

```diff
@@ -524,6 +524,22 @@ public DataOutputStream getDataOutputStreamUsingAppend(String path, FileFactory.
     return getFiles(listStatus);
   }
 
+  @Override public List<CarbonFile> listFiles(Boolean recursive, CarbonFileFilter fileFilter)
```

Comment:
Move ` @Override` down and add doc for public method

Reply:
ok